src/config/swagger.json

```
@@ -98,6 +98,66 @@
         }
       }
     },
+    "/users/reset": {
+      "post": {
+        "tags": ["auth"],
+        "summary": "Sends a registered user a password reset link",
+        "description": "This endpoint registers sends a registered user a link to reset their password",
+        "operationId": "resetPassword",
+        "consumes": ["application/json"],
+        "produces": ["application/json"],
+        "security": [],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Information needed send a user a password reset link",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/passwordReset"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/passwordResetResponse"
+            }
+          }
+        }
+      }
+    },
+    "/users/reset/:token": {
+      "patch": {
+        "tags": ["auth"],
+        "summary": "Updates a registered user password",
+        "description": "This endpoint updates the user password in the database",
+        "operationId": "updatePassword",
+        "consumes": ["application/json"],
+        "produces": ["application/json"],
+        "security": [],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Information needed to update the user password",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/updatePassword"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
```

You can change the status code here
```suggestion
          "200": {
```